dags.yaml

```diff
@@ -119,7 +119,7 @@ bqetl_accounts_backend_external:
   default_args:
     owner: akomar@mozilla.com
     start_date: "2023-09-19"
-    email: ["akomar@mozilla.com"]
+    email: ["akomar@mozilla.com", "telemetry-alerts@mozilla.com"]
```

Comment:
Now that notifications will be sent to `telemetry-alerts@mozilla.com`, should this DAG still be tagged `triage/no_triage`?